test/integration/xfcc_integration_test.cc

```diff
@@ -0,0 +1,243 @@
+#include "xfcc_integration_test.h"
```

Comment:
I've created https://github.com/lyft/envoy/issues/1162 to track, let's stick with what is there for now.